.travis/rpmbuild_test

```diff
@@ -5,13 +5,13 @@
 OS_VERSION="$1"
 
 # build and install
-yum install -y libtool rpm-build make yum-utils squashfs-tools
+yum install -y libtool rpm-build make yum-utils
 ./autogen.sh
 yum-builddep -y singularity.spec
 ./configure
 make dist
 rpmbuild -ta *.tar.gz
-rpm -iv ~/rpmbuild/RPMS/*/*.rpm
+yum install -y ~/rpmbuild/RPMS/*/*.rpm
```

Comment:
IMO, doing:  `yum localinstall -y ~/rpmbuild/RPMS/*/*.rpm` is better for an on disk location.

Reply:
Indeed! 
I would first wait for @c-ney who is basing his work on this PR if I understood correctly,
so I would not apply this change now since it may cause conflicts for him.
But this for sure should be changed to `localinstall` before the final version, thanks for the review! 👍

Reply:
From what I get from the man page of yum, there is no difference between `yum install file.rpm` and `yum localinstall file.rpm` [1]. Also the latter is _kinda_ marked as deprecated. But thinking about it, I did not check this in centos 6. The version there might not be new enough.

> If the name is a file, then install works like localinstall.

> Note that the install command will do a local install, if given a filename. This command is maintained for legacy reasons only.

[1] http://man7.org/linux/man-pages/man8/yum.8.html